cfme/infrastructure/host.py

```diff
@@ -214,6 +217,12 @@ def delete(self, cancel=True):
         cfg_btn(btn_name, invokes_alert=True)
         sel.handle_alert(cancel=cancel)
 
+    def load_details(self, refresh=False):
+        """To be compatible with the Taggable and PolicyProfileAssignable mixins."""
+        navigate_to(self, 'Details')
```

Comment:
It definitely needs some refactoring but bottom-line - we don't need 3 functions to do one thing.